portfolio/src/main/webapp/script.js

```diff
@@ -12,11 +12,36 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+var slideIndex = 1;
+
+// Next/previous controls
+function plusSlides(n) {
+  showSlides(slideIndex += n);
+}
+
+function showSlides(n) {
+  var i;
```

Comment:
You could declare the variable with the loop like for (var i = 0 ; i < slides.length; i ++) {...}

https://google.github.io/styleguide/jsguide.html#appendices-legacy-exceptions-var

Reply:
done!